diff --git a/node_modules/rn-country-picker/index.tsx b/node_modules/rn-country-picker/index.tsx
index 0935f1a..d580849 100644
--- a/node_modules/rn-country-picker/index.tsx
+++ b/node_modules/rn-country-picker/index.tsx
@@ -153,13 +153,13 @@ const CountryPicker = (props: CountryPickerProps) => {
 							}}
 						>
 							<Image
-								resizeMode="center"
+								resizeMode="contain"
 								style={styles.imageStyle}
 								source={props.backButtonImage}
 							/>
 						</Pressable>
 
-						{!hidePickerTitle && (
+						{/* {!hidePickerTitle && (
 							<Text
 								style={[
 									styles.pickerTitleStyle,
@@ -168,9 +168,9 @@ const CountryPicker = (props: CountryPickerProps) => {
 							>
 								{props.pickerTitle}
 							</Text>
-						)}
+						)} */}
 
-						{!hideSearchBar && (
+						{true && (
 							<TextInput
 								style={[
 									styles.searchBarStyle,
@@ -185,9 +185,9 @@ const CountryPicker = (props: CountryPickerProps) => {
 							/>
 						)}
 
-						<Pressable
+						{/* <Pressable
 							onPress={() => {
-								toggleSearchBar(!hideSearchBar);
+								// toggleSearchBar(!hideSearchBar);
 								togglePickerTitle(!hidePickerTitle);
 							}}
 						>
@@ -196,7 +196,7 @@ const CountryPicker = (props: CountryPickerProps) => {
 								style={styles.imageStyle}
 								source={props.searchButtonImage}
 							/>
-						</Pressable>
+						</Pressable> */}
 					</View>
 
 					<FlatList
@@ -364,7 +364,7 @@ const styles = StyleSheet.create({
 	},
 	imageStyle: {
 		width: 45,
-		height: 45,
+		height: 15,
 		transform: [{ scaleX: I18nManager.isRTL ? -1 : 1 }],
 	},
 	flatListStyle: {
